plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1608,7 +1608,7 @@ public void finishMerge(ConnectorSession session, ConnectorMergeTableHandle merg
         }
     }
 
-    private static void appendCdfFileEntries(
+    private static void appendCdcEntries(
             TransactionLogWriter transactionLogWriter,
             List<DataFileInfo> cdfFilesInfos,
```

Comment:
The method name was changed to use "cdc". I would rename to `cdcFilesInfos` including the method caller's argument.

Reply:
The method caller still uses `cdfFiles`.